lib/options.py

```diff
@@ -278,6 +278,18 @@ def __init__(self):
                    Enable long run tests.
                    """))
 
+        parser.add_argument(
+               "-l", "--log",
```

Comment:
I think no option is needed for this. Just enable logging and that's it.

Reply:
Removed it, thanks!